pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -697,8 +698,12 @@ servingInfo:
   keyFile: /var/serving-cert/tls.key
 providers: {}
 plugins:
-  pluginName1: https://serviceName1.serviceNamespace1.svc.cluster.local:8443/
-  pluginName2: https://serviceName2.serviceNamespace2.svc.cluster.local:8443/
+  enabled:
+    pluginName1: https://serviceName1.serviceNamespace1.svc.cluster.local:8443/
+    pluginName2: https://serviceName2.serviceNamespace2.svc.cluster.local:8443/
+    pluginName3: https://serviceName3.serviceNamespace3.svc.cluster.local:8443/
```

Comment:
This is a nit since it's only a test, but these aren't valid plugin names since they have uppercase letters. I'd make it `plugin1`, `plugin2`, `plugin3`. (Same for `serviceName`.)